frontend/src/queries/nodes/DataTable/ColumnConfigurator/ColumnConfigurator.tsx

```diff
@@ -109,9 +109,7 @@ function ColumnConfiguratorModal({ query }: ColumnConfiguratorProps): JSX.Elemen
             columnKey = column.substring(11)
         }
 
-        if (columnKey.includes('#')) {
-            columnKey = extractExpressionComment(columnKey)
-        }
+        columnKey = trimQuotes(columnKey)
```

Comment:
What I meant here is that it's probably a bug, and `#` should be replaced with `--`. Better yet, let's remove the `if()` part, as [this PR](https://github.com/PostHog/posthog/pull/15861) will eventually make `extractExpressionComment` also use the `alias` from `column as "alias"`. However we still want to show the comment as the column title if there is one.